src/utils/analytics/events.ts

```diff
@@ -173,10 +173,10 @@ export const activityExternalLinkClick = (
 /* We implement different CTAs to invite the user to intentinally go to a course or any type of content. This event will help us understand  how CTA's do on different pages */
 export const activityCtaClick = (
   resourceType: string,
-  instructor: string,
   currentLocation: string,
-  topic: string,
-  redirectTo: string,
+  instructor?: string,
+  topic?: string,
+  redirectTo?: string,
```

Comment:
Updates analytics event to be more flexible. A lot of the times we just care about current location and resourceType